mu/harness/sub_sim.py

```diff
@@ -0,0 +1,6 @@
+class SubSim:
```

Comment:
I did think about this, but I remember a long way back you mentioned a remote possibility of having MPPT in non-driver code call into MU, so I figured call it a subsim in case we need to have subsubsims.

made it an abstract base class.